tasks.rb

```diff
@@ -185,6 +190,8 @@ def create_osws
     'base-hvac-furnace-wood-only.osw' => 'base.osw',
     # 'base-hvac-furnace-x3-dse.osw' => 'base.osw', # Not going to support DSE
     'base-hvac-ground-to-air-heat-pump.osw' => 'base.osw',
+    # 'base-hvac-ground-to-air-heat-pump-cooling-only.osw' => 'base-hvac-ground-to-air-heat-pump.osw', # FIXME
```

Comment:
What's the problem with this one?

Reply:
https://github.com/NREL/OpenStudio-HPXML/blob/master/workflow/sample_files/base-hvac-ground-to-air-heat-pump-cooling-only.xml#L313

Reply:
Ah, that's a bug. Will fix it.

Reply:
https://github.com/NREL/OpenStudio-HPXML/pull/611